doc/api/buffer.markdown

```diff
@@ -577,6 +577,46 @@ for (var key of buf.keys()) {
 //   5
 ```
 
+### buf.lastIndexOf(value[, byteOffset][, encoding])
+
+* `value` {String}, {Buffer} or {Number}
+* `byteOffset` {Number} Default: `buf.length`
+* `encoding` {String} Default: `'utf8'`
+* Return: {Number}
+
+Identical to [`Buffer#indexOf()`][], but searches the Buffer from back to front
+instead of front to back. Returns the starting index position of `value` in
+Buffer or `-1` if the Buffer does not contain `value`. The `value` can be a
+String, Buffer or Number. Strings are by default interpreted as UTF8. If
+`byteOffset` is provided, will return the last match that begins at or before
+`byteOffset`.
+
+```js
+const buf = new Buffer('this buffer is a buffer');
```

Comment:
@tflanagan no other code sample in this doc does that

which linter are you talking about? `make test` and `make doc` both run without errors